sixMans/game.py

```diff
@@ -11,12 +12,11 @@
 class Game:
     def __init__(
             self, players, queue: SixMansQueue,
-            guild: discord.Guild=None,
```

Comment:
I think having the guild in the game class can be helpful, but I don't think it is an issue either way.